storage/oasis/nodeapi/file/kvstore_test.go

```diff
@@ -0,0 +1,111 @@
+package file
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/oasisprotocol/oasis-indexer/log"
+	"github.com/stretchr/testify/require"
+)
+
+// Returns an open KVStore, and a function to clean it up.
+func openTestKVStore(t *testing.T) (KVStore, func()) {
+	const path = "/tmp/indexer-kv-test"
```

Comment:
Small nit:
```suggestion
	path, err := os.MkdirTemp("", "indexer-kv-test")
```

to ensure that the tests won't use a potentially left-over data if a previous test run process exited uncleanly and didn't got to clean the folder.